app/styles/power-select.less

```diff
@@ -0,0 +1,36 @@
+.upf-power-select{
+  height: 100%;
+
+  .upf-power-select__array-container {
+    position: relative;
+    height: 100%;
+    .border;
+    .border-color-default;
+    .border-radius-default;
+
+    .infinite-shadow-scroll {
+      position: absolute;
+      content: '';
```

Comment:
`content` set to empty is needed for the effect to show right ?

Reply:
Yes!